internal/host/validator.go

```diff
@@ -305,6 +305,57 @@ func (v *validator) isMediaConnected(c *validationContext) (ValidationStatus, st
 	}
 }
 
+// isMtuOKAllInterfaces - this validation is intended for UMN, where the MTU report is checked across all interfaces.
+func (v *validator) isMtuOKAllInterfaces(c *validationContext) (ValidationStatus, string) {
+	if c.infraEnv != nil {
+		return ValidationSuccessSuppressOutput, ""
+	}
+
+	connectivityReport, err := hostutil.UnmarshalConnectivityReport(c.host.Connectivity)
+	if err != nil {
+		return ValidationError, ""
```

Comment:
Is there a reason to hide the failure reason in this case (and below when it's not a ValidationSuccessSuppressOutput) ? If this information could be display to the end user, we shouldn't display the internal error (but it would be convenient to understand the problem, too bad to just lose it), but we can have something that gives hints where the check fail at least